Essentials/src/main/java/com/earth2me/essentials/commands/Commandeditsign.java

```diff
@@ -60,17 +81,13 @@ protected void run(final Server server, final User user, final String commandLab
 
                 if (line == -1) {
                     for (int i = 0; i < 4; i++) {
-                        processSignCopyPaste(user, sign, i, copy);
+                        processSignCopyPaste(user, target, sign, i, copy);
```

Comment:
This will result in one `SignChangeEvent` fired per line when pasting an entire sign. Could move this `if` down into the process method?

Reply:
correct, i thought about this but decided that I don't care. Moving around this would introduce more code complexity/repetition for what gain? do we really care that the event is fired more than once. Technically, we are modifying the block more than once and thus causing more than one block update anyway.